deploy/docs/additional_prometheus_configuration.md

```diff
@@ -149,70 +149,54 @@ prometheus-operator:  # For values.yaml
+To do that, you should add an output to your values.yaml:
+
 ```yaml
-# ...
-spec:
+
+fluentd:
   # ...
-  template:
+  metrics:
     # ...
-    spec:
+    output:
       # ...
-      containers:
-        # ...
-        - env:
-          # ...
-          - name: SUMO_ENDPOINT_LOGS
-            valueFrom:
-              secretKeyRef:
-                key: endpoint-logs
-                name: sumologic
-          - name: MY_CUSTOM_METRICS
-            valueFrom:
-              secretKeyRef:
-                key: my-custom-metrics
-                name: sumologic
-          - name: LOG_FORMAT
-            value: fields
+      my_source:  # It matches sumologic.sources.my_source
+        tag: prometheus.metrics.YOUR_TAG  # tag used by Fluentd's match clausule
+        id: sumologic.endpoint.metrics
+      
+      # alternative example source with all fields
+      alternative_source:  # It doesn't match any source name
+        tag: prometheus.metrics.YOUR_TAG**
+        id: sumologic.endpoint.metrics
+        source: my_source  # you can specify which source should be used (the output key (alternative_source) is taken by default)
+        weight: 10  # optional, by default takes 0 and it is use to prioritise outputs (less means more important)
+        drop: true  # Drop records which match the tag (false by default)
 ```
 
-  * Finally, you need to modify the Fluentd config to route data to your newly created HTTP source. Assuming you installed the collector in the `sumologic` namespace, you can run `kubectl -n sumologic edit configmap fluentd` or edit the YAML you deployed when you set up collection. Note, if you installed using helm, the name of the deployment may be different depending on how you installed the helm chart.
-  * Locate the section `match prometheus.metrics` and you will insert a new section above this. The `match` statement should end with a tag that identifies your data that Fluentd will use for routing. Then make sure you point to the environment variable you added to your deployment. The following is an example.
-  
-```
-...        
-          <match prometheus.metrics.YOUR_TAG>
-             @type sumologic
-             @id sumologic.endpoint.metrics
-             endpoint "#{ENV['MY_CUSTOM_METRICS']}"
-             @include metrics.output.conf
-           </match>
-          <match prometheus.metrics**>
-             @type sumologic
-             @id sumologic.endpoint.metrics
-             endpoint "#{ENV['SUMO_ENDPOINT_METRICS']}"
-             @include metrics.output.conf
-           </match>
-```
+The weight concept helps to keep Fluentd outputs in the correct order.
+In the above example, the `alternative_source` covers all tags for the `my_source`.
+This means if `alternative_source` has less weight than `my_source`
+it would process all records and none of them would be taken by `my_source`.
```

Comment:
This could be a difficult concept for those unfamiliar with Fluentd pipelines, maybe a link to some Fluentd documentation could be helpful here